test/integration/scss/test/index.test.js

```diff
@@ -34,17 +34,20 @@ describe('SCSS Support', () => {
         env: { NODE_OPTIONS: shellQuote([`--require`, mockFile]) },
         stderr: true,
       })
+      let cleanScssErrMsg =
+        '\n\n' +
+        './styles/global.scss\n' +
+        "To use Next.js' built-in Sass support, you first need to install `sass`.\n" +
+        'Run `npm i sass` or `yarn add sass` inside your workspace.\n' +
+        '\n' +
+        'Learn more: https://err.sh/next.js/install-sass\n' +
+        '\n'
+
       expect(code).toBe(1)
       expect(stderr).toContain('Failed to compile.')
-      expect(stderr).toContain(
-        "To use Next.js' built-in Sass support, you first need to install `sass`."
-      )
-      expect(stderr).toContain(
-        'Run `npm i sass` or `yarn add sass` inside your workspace.'
-      )
-      expect(stderr).toContain(
-        'Learn more: https://nextjs.org/docs/messages/install-sass'
-      )
+      expect(stderr).toContain(cleanScssErrMsg)
+
+      expect(stderr).not.toContain('Require stack:')
```

Comment:
```suggestion
```

This is not necessary anymore.

Reply:
resolved in commit [7086fa0](https://github.com/vercel/next.js/pull/35051/commits/7086fa04cd5d83afc8bb38dd6949292b546a3312)